src/pages/Dashboard/cards/OracleStatusCard/index.tsx

```diff
@@ -18,6 +18,7 @@ import Stats, {
 import ErrorFallback from 'components/ErrorFallback';
 import { StoreType } from 'common/types/util.types';
 import Ring64, {
+  Ring64Pair,
```

Comment:
```suggestion
  Ring64Subtitle
```

Getting an import error which is causing the build to fail. Did fix this, but couldn't push to your fork :)